app/src/androidTest/java/de/test/antennapod/ui/UITestUtils.java

```diff
@@ -113,6 +113,15 @@ private File newMediaFile(String name) throws IOException {
         return mediaFile;
     }
 
+    /**
+     * Update feed at the given index on the server.
+     *
+     * @param feedIndex the index of feed to update in `hostedFeeds`.
+     */
+    public void refreshFeedOnServer(int feedIndex) throws IOException {
+        hostFeed(hostedFeeds.get(feedIndex));
```

Comment:
I don't really understand what this does... Does it take changes that external callers have applied to its internal state and applies them? That feels quite weird. I would instead give it a feed as parameter.

Reply:
Yes, my goal is to update a feed that is already hosted on the server. By modifying it on the server, I can confirm the feed is refreshed after the update to authentication details. If you have a better way of checking that the update occurs, I am all ears!

I was originally passing in a feed, but went this direction as I wanted to make sure you couldn't host a feed that is not in `hostedFeeds`. If I pass a feed as the parameter, would you want to make sure it is in `hostedFeeds`, or leave that up to the tester? If we aren't concerned about doing that here, could we just make `hostFeed` public?

Reply:
Hmm, good question. Not entirely sure what I like better, but I guess it's okay like it is now.